cmd/dtbs/dtbs.go

```diff
@@ -0,0 +1,267 @@
+package databaseshortener
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	bn "go-prof-sprint-1/cmd/bindata"
+
+	flw "go-prof-sprint-1/cmd/flwrk"
+
+	"github.com/golang-migrate/migrate"
+	"github.com/golang-migrate/migrate/database/sqlite3"
+	bindata "github.com/golang-migrate/migrate/source/go_bindata"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/pkg/errors"
+)
+
+func NewDB(dbPath string) (*sql.DB, error) {
+	sqliteDB, err := sql.Open("sqlite3", dbPath)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to open sqlite DB")
+	}
+
+	return sqliteDB, nil
+}
+
+func RunMigrateScripts(db *sql.DB) error {
+	driver, err := sqlite3.WithInstance(db, &sqlite3.Config{})
+	if err != nil {
+		return fmt.Errorf("creating sqlite3 db driver failed %s", err)
+	}
+
+	res := bindata.Resource(bn.AssetNames(),
+		func(name string) ([]byte, error) {
+			return bn.Asset(name)
+		})
+
+	d, _ := bindata.WithInstance(res)
+	m, err := migrate.NewWithInstance("go-bindata", d, "sqlite3", driver)
+	if err != nil {
+		return fmt.Errorf("initializing db migration failed %s", err)
+	}
+	_ = m.Down()
+	err = m.Up()
+	if err != nil && err != migrate.ErrNoChange {
+		return fmt.Errorf("migrating database failed %s", err)
+	}
+
+	return nil
+}
+
+func DataBaseMainPageCfg() (vbn_ string, err error) {
+	var db *sql.DB
+	var vbn string
+	dbName := "shortener.db"
+	db, err = sql.Open("sqlite3", dbName)
+	if err != nil {
+		return "", err
+	}
+	defer db.Close()
+	quer := "SELECT vbn FROM cfg WHERE id = 1;"
+	rows, err := db.Query(quer)
+	if err != nil {
+		return "", err
+	}
+	defer rows.Close()
+	if rows.Err() != nil {
+		return "", rows.Err()
+	}
+	rows.Next()
+	err = rows.Scan(&vbn)
+	if err != nil {
+		return "", err
+	}
+	return vbn, nil
+}
+
+func DatBaseAPIPageGet(id string) (longURL_ string, flag int, err error) {
```

Comment:
CHANGE REQUEST: please read Go coding style and rename all variables according to it.